### PR TITLE
Fix for General download/upload speed cannot be set in preferences #56.

### PR DESCRIPTION
**Pull request by snaunton**
#56.

I can't find any spinners that actually represent doubles, so just changed it to use ints instead

Have not found any side effects.

### CODE REVIEW

File: src/trg-json-widgets.c [hunk 1]
```diff
@@ -121,16 +123,52 @@ GtkWidget *trg_json_widget_entry_new(GList ** wl, JsonObject * obj,
...
+}
+
+GtkWidget *trg_json_widget_spin_double_new(GList ** wl, JsonObject * obj,
                                     const gchar * key,
                                     GtkWidget * toggleDep, gdouble min,
                                     gdouble max, gdouble step)
 {
     GtkWidget *w = gtk_spin_button_new_with_range(min, max, step);
+
+    gtk_spin_button_set_digits(GTK_SPIN_BUTTON(w),
```
Comment (TingPing) [inline src/trg-json-widgets.c]: Looks like this used to be 1 digit? Any reason for changing it?

**Comment by TingPing** [comment]
Ugh, overlooked that empty commit message..

**Comment by snaunton** [comment]
I know transmission-remote command line a lot better than transmission-remote-gtk and know that you can enter, for instance, 1.25 for seed ratio. So just replicated that behavior without too much thought, and 3 digits seemed like too much. Not attached to this so, new commit which replicates old behavior.

**Comment by TingPing** [comment]
Fair enough. I didn't really have anything against it either just wondered why.

**Pull request merged**